src/NServiceBus.Core.Tests/Recoverability/When_message_is_deferred_by_slr.cs

```diff
@@ -0,0 +1,145 @@
+﻿namespace NServiceBus.Core.Tests.Recoverability
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+    using NServiceBus.Pipeline;
+    using NServiceBus.Transports;
+    using NUnit.Framework;
+    using Testing;
+
+    [TestFixture]
+    public class When_message_is_deferred_by_slr
+    {
+        FakeDispatchPipeline dispatchPipeline;
+
+        [SetUp]
+        public void SetUp()
+        {
+            dispatchPipeline = new FakeDispatchPipeline();
+        }
+
+        [Test]
+        public async Task _should_abort_message_receive_when_marking_for_deferral()
```

Comment:
I'm not sure (@weralabaj any ideas?).  That being said I've changed that to be consistent with the rest of the tests.